.github/ISSUE_TEMPLATE/bug-task.yaml

```diff
@@ -0,0 +1,77 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
```

Comment:
I think this should be "saw" instead of "see" :)

Reply:
Was really tempted to add a dropdown for real world vs VR world.

Reply:
Hahaha, yes indeed. :)

PS. this should be updated to saw (consistency of tenses between "what happened" and "what you saw")